### PR TITLE
added OpenLDAP beta integration

### PR DESCRIPTION
**Pull request by IkaDemisto**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/14091

## Description
Added OpenLDAP Beta authentication integration.

## Required version of Demisto
5.0.0

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [x] Code Review

## Dependencies
Tests:
- [x] Test-IsMaliciousIndicatorFound
- [x] JSONtoCSV-Test
- [x] ActiveMQ Test

## Additional changes
https://github.com/demisto/server/pull/14059

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

**Comment by IkaDemisto** [comment]
I will add unit tests later

File: Beta_Integrations/OpenLDAP/OpenLDAP.yml [hunk 1]
```diff
@@ -0,0 +1,156 @@
+category: Authentication
+commonfields:
+  id: OpenLDAP
+  version: -1
+configuration:
+  - display: Fetch incidents
```
Comment (Itay4) [inline Beta_Integrations/OpenLDAP/OpenLDAP.yml]: fetch incidents params should be the bottom ones

File: Beta_Integrations/OpenLDAP/OpenLDAP.yml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    type: 13
+  - display: Server IP or Host Name (e.g. 192.168.0.1)
+    name: host
+    required: true
+    type: 0
+  - display: Port. If not specified, default the port is 389, or 636 for LDAPS.
+    name: port
+    required: false
+    type: 0
+  - display: Credentials
```
Comment (Itay4) [inline Beta_Integrations/OpenLDAP/OpenLDAP.yml]: lets find better display name

File: Beta_Integrations/OpenLDAP/OpenLDAP.yml [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    type: 0
+  - defaultvalue: None
+    display: Connection Type
+    name: connection_type
+    options:
+      - None
+      - SSL
+    required: true
+    type: 15
+  - defaultvalue: 'false'
```
Comment (Itay4) [inline Beta_Integrations/OpenLDAP/OpenLDAP.yml]: no need for defaultValue

File: Beta_Integrations/OpenLDAP/OpenLDAP.yml (hunk 2 above)
Comment (IkaDemisto) [reply Beta_Integrations/OpenLDAP/OpenLDAP.yml]: @Itay4 you mean instead of `Credentials`?

**Comment by IkaDemisto** [comment]
@dorsha @anara123 @yaakovi **FYI** `OpenLDAP` integration also works and authenticates with Active Directory instance (with non default filters).

**Pull request merged**